bedrock/firefox/templates/firefox/family/index-b.html

```diff
@@ -0,0 +1,173 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "base-resp.html" %}
```

Comment:
Should this extend `firefox/base-resp.html` instead? Currently the page is getting the new global nav, which seems out of place with the family navigation.